scalafix/build.sbt

```diff
@@ -57,3 +64,47 @@ lazy val tests = project
   )
   .dependsOn(rules)
   .enablePlugins(ScalafixTestkitPlugin)
+
```

Comment:
Created other sets of module for the scalafix v2, as it uses different version of fs2. To run the v2 tests, run `sbt tests2/test`